tests/server-multiple-retention-policies/README.md

```diff
@@ -0,0 +1,222 @@
+* **AC2**: The versionbased retention policy only retains the x given newest versions of each state-object.
+* **AC3**: The point in time retention policy removes all members that are older than a given point in time.
+* **AC4**: I can configure multiple retention policies
+* **AC5**: The LDES server automatically removes all members that match ALL retention policies
+
+## Test Scenario
+
+For each retention policy, we create a view with this retention.
+We then send some test members to the LDES server and check if the desired number of members remain in the database.
+
+### Start Systems
+
+When started, we expect that the LDES server creates the initial state: the LDES itself with a collection with no views.
+
+### Add the view and sending data
+
+We add a paginated view with the retention policy we wish to test.
+We then start sending a new member to the server every second.
+
+### Observing member count
+
+When now continuously poll the member count in the LDES server throughout a given period and observe if this is in line with the expected outcome.
+
+### Test teardown
+
+To stop the test, we stop sending data, remove the view and tear down the docker containers
+
+## Test Setup
+> **Note**: if needed, copy the [environment file (.env)](.env) to a personal file (e.g. `user.env`) and change the settings as needed. If you do, you need to add ` --env-file user.env` to each `docker compose` command.
+
+Run all systems except the workflow by executing the following (bash) command:
+```bash
+docker compose up -d
+```
+Please ensure that the LDES Server is ready to ingest by following the container log until you see the following message `Cancelled mongock lock daemon`:
+```bash
+docker logs --tail 1000 -f $(docker ps -q --filter "name=ldes-server$")
+```
+Press `CTRL-C` to stop following the log.
+
+Seed the server with a collection.
+```bash
+curl -X POST 'http://localhost:8080/admin/api/v1/eventstreams' -H 'Content-Type: text/turtle' -d '@./config/mobility-hindrance.ttl'
+```
+
+Remove the default view.
+```bash
+curl -X DELETE 'http://localhost:8080/admin/api/v1/eventstreams/mobility-hindrances/views/by-page'
+```
+
+Verify the initial collection is created:
+```bash
+curl -i http://localhost:8080/mobility-hindrances
+```
+
+## Test Timebased Retention
+To execute this test scenario, run the following steps:
+
+1. Add the timebased view:
+   ```bash
+   curl -X POST 'http://localhost:8080/admin/api/v1/eventstreams/mobility-hindrances/views' -H 'Content-Type: text/turtle' -d '@./config/view-timebased.ttl'
+   ```
+
+2. Start sending data to the LDES server:
+    ```bash
+    docker compose up ldes-message-generator
+    ```
+
+3. Repeatedly check the number of members in the database:
+   ```bash
+   bash ./config/poll-members.sh
+   ```
+   The amount of members should steadily increase until it stagnates between 30 and 40 members.
+
+4. Stop sending data:
+    ```bash
+    docker stop basic-retention_member-generator
```

Comment:
On line 72, you start this container using docker compose and another name. This can be quite confusing.
Why don't you issue something like:
`docker compose rm -s -f -v test-message-generator`

Note that you also don't need the prefix 'basic-retention' as that can be changed with a variable, it would break your commands.